fluid-client/examples/24k.js

```diff
@@ -0,0 +1,99 @@
+const fluid = require('../src/fluidOsc');
+const FluidClient = require('../src/FluidClient');
+const tab = require('../src/tab');
+const path = require('path');
+const drums909 = require('../recipes/track-drums909');
+const drumTrackName = '909';
+const sampleDir =  path.join(__dirname, '909kit');
+const drumsMsg = drums909(sampleDir, drumTrackName);
+
+const bassline_bar1 = {
+    notes:['f3','e3', 'a3','c4'],
+    rhythm: '1e+a2e+a3e+a4+',
+    pattern:'0---...1002-32',
+};
+
+const build =   '0-..1-..2-......'
+const BbChord = '0-.0............'
+const FmChord = '3-.3............'
+const Bb_extended_Chord='0-.0....4-..5-..'
+const chorus_notes = {
+    noteLibrary: [
+        ['bb4', 'c#5', 'f5'], //Bbm
```

Comment:
I think this should be `['Bb4', 'Db5', 'F5', 'Bb5'] (since its a Bb chord, it should have Db, not C3) It also sound like it has the octave at the top top me